operator/main.go

```diff
@@ -61,6 +61,10 @@ var (
 	// see https://github.com/kubernetes-sigs/controller-runtime/blob/v0.8.3/pkg/webhook/server.go#L96-L104
 	defaultCertDir  = filepath.Join(os.TempDir(), "k8s-webhook-server", "serving-certs")
 	defaultTLSPaths = []string{filepath.Join(defaultCertDir, "tls.crt"), filepath.Join(defaultCertDir, "tls.key")}
+	// centralLabelSelector is a kubernetes label selector that is used to filter out Central instances
+	// to be managed by this operator. If the selector is empty, all Central instances are managed.
+	// see https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
+	centralLabelSelector = env.RegisterSetting("CENTRAL_LABEL_SELECTOR", env.WithDefault(""))
```

Comment:
Typically all StackRox related env vars are prefixed with `ROX_`, acutally I do not recall whether this also applies to the operator. Could you check if other env vars using this prefix?
I think this might be the first operator-specific env var which has no required naming schema.

```suggestion
	centralLabelSelector = env.RegisterSetting("ROX_CENTRAL_LABEL_SELECTOR", env.WithDefault(""))
```